server.go

```diff
@@ -215,7 +215,15 @@ func getCombinedRouting(endpoints []string, dht routing.Routing) (router, error)
 	var routers []router
 
 	for _, endpoint := range endpoints {
-		drclient, err := client.New(endpoint, client.WithUserAgent(userAgent))
+		drclient, err := drclient.New(endpoint,
+			drclient.WithUserAgent("someguy/"+buildVersion()),
+			drclient.WithProtocolFilter([]string{
+				"unknown", // allow results without protocol list, allowing end user to do libp2p Identify probe to test them
+				"transport-bitswap",
+				"transport-ipfs-gateway-http",
+			}),
```

Comment:
For what someguy is doing (i.e. serving as a proxy) why not let all protocol through rather than limiting them?

e.g. why shouldn't GraphSync peers, or those from some new protocol show up here without being explicitly added?

Given that programmatic clients that want to use the data for content routing directly (e.g. helia) will send their own filters to  be layered on top pre-filtering the data doesn't seem necessary.

Reply:
Good point about no filtering being a better default for someguy and ecosystem.
Disabled filtering in https://github.com/ipfs/someguy/pull/85/commits/a2abe25feaf8f6f025f65fdb4e5d6e86a384f027.


But I've filled https://github.com/ipfs/someguy/issues/86 to add support for setting implicit filters for deployments that would benefit from reduced default response sizes.